static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -197,25 +197,33 @@ export function getOnboardingTasks({
       location: getOnboardingInstructionsUrl({projects, organization}),
       display: true,
       SupplementComponent: withApi(
-        ({api, task, onCompleteTask}: FirstEventWaiterProps) =>
-          !!projects?.length &&
-          task.requisiteTasks.length === 0 &&
-          !task.completionSeen ? (
+        ({api, task, onCompleteTask}: FirstEventWaiterProps) => {
+          if (hasQuickStartUpdatesFeature(organization)) {
+            if (!projects?.length || task.requisiteTasks.length > 0 || taskIsDone(task)) {
+              return null;
+            }
+            return (
+              <EventWaitingIndicator
+                text={t('Waiting for error')}
+                hasQuickStartUpdatesFeature
+              />
+            );
+          }
+
+          return !!projects?.length &&
+            task.requisiteTasks.length === 0 &&
+            !task.completionSeen ? (
```

Comment:
`task.completionSeen` was a left over of https://github.com/getsentry/sentry/pull/79666